src/GraphQL/Transports/Json/WebSocketMessage.cs

```diff
@@ -0,0 +1,24 @@
+namespace GraphQL.Transports.Json
+{
+    public class WebSocketMessage
```

Comment:
```suggestion
    public class OperationMessage
```

The class name must reflect its purpose, and not the information transfer channel. Messages of this type do not have to be sent through websockets as well as `GraphQLRequest` objects do not have to be sent through http calls - they **may** be sent through websockets too. In general they all **may** be sent through any transport.

Reply:
True, although this class is designed for a specific transport, namely `graphql-ws` or `graphql-transport-ws`, which is WebSocket over HTTP/S.  Notice the namespace.  Are you sure this should be changed?

Reply:
Yes. I believe that `OperationMessage` better reflects the meaning, and the `WebSocketMessage` only imposes unnecessary obligations.

Reply:
Done